sdk/python/kubeflow/storage_initializer/hugging_face.py

```diff
@@ -39,6 +39,7 @@ class HuggingFaceModelParams:
     model_uri: str
     transformer_type: TRANSFORMER_TYPES
     access_token: str = None
+    num_labels: Optional[int] = None
```

Comment:
Did you check if we don't need to set this parameter when we download the model: https://github.com/kubeflow/training-operator/blob/c0406d43b407ac86ec134eae6a3d19bba55ad1df/sdk/python/kubeflow/storage_initializer/hugging_face.py#L70 ?

Reply:
@andreyvelich Do you mean test the example without setting "num_labels"? I tried not setting "num_labels" using [the example from the tutorial](https://www.kubeflow.org/docs/components/training/user-guides/fine-tuning/), and it still produced the same error, `IndexError: Target 4 is out of bounds,` in the CPU environment, which proves that there is no problem with handling "num_labels" when it's not set.

Reply:
@andreyvelich Oh, sorry, I misunderstood. I have tested it, and we don't need to set `num_labels` in the Model Provider in storage initializer.